pkg/controller/plan/validation.go

```diff
@@ -1837,3 +1842,83 @@ func (r *Reconciler) IsValidTargetName(targetName string) error {
 
 	return nil
 }
+
+func (r *Reconciler) validateVddkAndOffloadMixedUsage(plan *api.Plan) error {
+	source := plan.Referenced.Provider.Source
+	if source == nil {
+		return nil
+	}
+
+	if source.Type() != api.VSphere {
+		return nil
+	}
+
+	if !settings.Settings.Features.CopyOffload {
+		return nil
+	}
+
+	mixedUsage := libcnd.Condition{
+		Type:     VDDKAndOffloadMixedUsage,
+		Status:   True,
+		Reason:   NotSupported,
+		Category: api.CategoryCritical,
+		Message:  "VM has mixed disk migration methods (VDDK and copy offload). All disks in a VM must use the same migration method.",
```

Comment:
Users likely not understand immediately that the method is driven by the storage map. Lets help them by referring them to that storage map